superset-frontend/src/assets/stylesheets/less/variables.less

```diff
@@ -184,7 +184,7 @@
 }
 
 // ****************************** Families ******************************
-@font-family-sans-serif: 'Inter', Helvetica, Arial;
+@font-family-sans-serif: 'Inter', Helvetica, Arial, 'Vazirmatn';
```

Comment:
The PR doesn't describe the motivation for the fonts in general, but it's probably worth noting that Inter, Helvetica, and Arial will override Vzairmatn in this case, and we're still loading the Inter bundles, so this might just be making the payload/bundle larger for no good reason.